src/components/AddMatches.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import '../App.css';
+
+import GamesList from '../components/GamesList';
+
+import { connect } from 'react-redux';
+import { StoreState } from '../types/index';
+import AutoComplete from 'material-ui/AutoComplete';
+
+const style: any = {
+  display: 'block',
+  margin: '0 auto'
+};
+const mapStateToProps = (state: StoreState) => ({
+  gamesList: state.gamesList
+});
+
+// Later this will take dispatch: any as argument
+const mapDispatchToProps = () => ({});
+
+const GamesListContainer = connect(mapStateToProps, mapDispatchToProps)(
+  GamesList
+);
+
+class AddMatches extends React.Component {
```

Comment:
As above  connect is used to pass data to GamesList using mapStateToProps. In the same way, create a container on top of your AddMatches component using connect and use that container everywhere.

Further, access that value from state inside AddMatches component by using `this.props.gamesList` for example.